index.html

```diff
@@ -0,0 +1,151 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8">
+    <title>Landing Page</title>
+    <link rel="stylesheet" href="css/style.css">
+  </head>
+
+  <body>
+   <div id="header">
+     
+    <img src="images\original_trombones.png" alt="company-logo" id="header-img">
+     <nav id="nav-bar">
+
+     <a href="#" class="nav-link">features</a>
+     <a href="#" class="nav-link">how it works</a>
+     <a href="#" class="nav-link">pricing</a>
```

Comment:
- [ ] Great job so far with the Navbar. Let's perfect it by adding links to the different page sections. This will ease the accessibility of the website. 😄